package.json

```diff
@@ -16,7 +16,8 @@
   },
   "scripts": {
     "start": "yarn update-abis; parcel public/dashboard.html public/widget.html",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "ganache_fork": "ganache-cli -f $(node infura_key.js) -m \"$(node mnenomic.js)\"",
```

Comment:
@brianwchou maybe you can help me find a better way to grab the key and mnenomic